src/controller/java/src/matter/controller/model/Paths.kt

```diff
@@ -54,6 +87,7 @@ data class CommandPath(
   val endpointId: UShort,
   val clusterId: UInt,
   val commandId: UInt,
+  val groupId: Optional<UInt> = Optional.empty()
```

Comment:
use UInt? since Kotlin have  default support for nuallble field

Reply:
I will change it to UInt? .

Reply:
Yes, we've been using an incompatible Optional package internally, which has caused us significant issues. We aim to fully decouple from the Optional package in Kotlin API.